zone/mob.h

```diff
@@ -1025,8 +1025,8 @@ class Mob : public Entity {
 	uint8 GetFlurryChance() { return GetSpecialAbilityParam(SPECATK_FLURRY, 0); }
 
 	static uint32 GetAppearanceValue(EmuAppearance in_appearance);
-	void SendAppearancePacket(uint32 type, uint32 value, bool whole_zone = true, bool ignore_self = false, Client* target = nullptr);
-	void SetAppearance(EmuAppearance app, bool ignore_self = true);
+    void SendAppearancePacket(uint32 type, uint32 value, bool WholeZone = true, bool iIgnoreSelf = false, Client* specific_target = nullptr, bool WholeZoneMinusMyGuild = false);
```

Comment:
Why were these reverted to non `snake_case`?

Reply:
The function is under review elsewhere and will update once finalized.

Reply:
New updates here for review